src/game/g_cmds.c

```diff
@@ -3191,6 +3191,16 @@ qboolean Cmd_CallVote_f(gentity_t *ent, unsigned int dwCommand, int fRefCommand)
 				CP("cp \"You cannot call a vote as a spectator.\"");
 				return qfalse;
 			}
+			else if (g_gamestate.integer == GS_WARMUP_COUNTDOWN && (level.warmupTime - level.time) < VOTE_TIME)
+			{
+				CP("cp \"You cannot call a vote when warmup is about to end.\"");
+				return qfalse;
+			}
+			else if (g_gamestate.integer == GS_PLAYING && (level.startTime + (g_timelimit.value * 60000) - level.time < VOTE_TIME))
+			{
+				CP("cp \"You cannot call a vote when game is about to end.\"");
+				return qfalse;
+			}
```

Comment:
I would probably revert this change and keep the old behavior for end of game scenario, at least in stopwatch mode. When playing a map with dynamite objective, players might call surrender or match reset to get over with the game at that point, which would not be possible after this.

Reply:
Thanks for the feedback and the review! Makes a lot of sense!